packages/ui/src/components/modals/Send.tsx

```diff
@@ -119,24 +129,30 @@ const Send = ({ onClose, className, onSuccess, onFinalized }: Props) => {
     [getMultisigByAddress, selectedMultiProxy]
   )
 
-  const easySetupOptions: { [index: string]: ReactNode } = useMemo(() => {
+  const easySetupOptions: Record<EasyTransferTitle, ReactNode> = useMemo(() => {
     return {
-      [SEND_TOKEN_MENU]: (
+      'Send tokens': (
```

Comment:
With enum, it will be easier to use the value without specifying a string

Reply:
This is no a string, this is a `EasyTransferTitle`. You can't put any string in there without a type error.

Reply:
Yeah, I saw the type, it's still confusing to read. I could understand if you created a type with one-word strings: like active | disabled | focus and used them this way. But for a 2-3 word string, it's probably the case for an enum where you don't use directly the value but the link to it. You can use the whole sentence as a type, but it doesn't mean you should do it. It works both ways, but in my opinion with an enum, it gives better readability and expectation that this is a predefined value.